### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**

### CODE REVIEW

File: cinema/serializers.py [hunk 1]
```diff
@@ -1 +1,96 @@
...
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "title", "description", "duration", "actors", "genres")
+
+
+class MovieListSerializer(MovieSerializer):
+    actors = serializers.SerializerMethodField()
+    genres = serializers.SerializerMethodField()
```
Comment (YuriiKindrat) [inline cinema/serializers.py:50]: u can use StringRelatedSerializer here instead

File: cinema/views.py [hunk 2]
```diff
@@ -1 +1,68 @@
...
+        if self.action == "retrieve":
+            return MovieDetailSerializer
+        return MovieSerializer
+
+
+class MovieSessionViewSet(viewsets.ModelViewSet):
+    queryset = MovieSession.objects.all().prefetch_related(
+        "movie",
+        "cinema_hall"
+    )
```
Comment (YuriiKindrat) [inline cinema/views.py:60]: no need to use `all()` mehtod before joins
also are u sure about prefetch_related for moviesessions ?

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: cinema/views.py [hunk 3]
```diff
@@ -1 +1,68 @@
...
+        return ActorSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all().prefetch_related("genres", "actors")
```
Comment (LiudmylaKulinchenko) [inline cinema/views.py:45]: no need to use all() method before joins

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done 🎈👏

**Pull request closed without merging**